app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self.connection.commit()
+
+    def all(self) -> list:
+        cursor = self.connection.execute(
+            f"SELECT * FROM {self.table_name}"
```

Comment:
Similar to the previous comment, ensure that the table name is not user-controlled to prevent SQL injection vulnerabilities.